add more emotions into ascii art

phamquochoan opened the pull request.

Kapeli commented:
Why did you delete some of the categories and emoticons?

Also, this build fails. Please see https://travis-ci.org/Kapeli/cheatsheets/jobs/51788637.

phamquochoan commented:
Well, in fact, I deleted 1 emotion, the `name 'Rock On!'` one because it's multiple lines and I think it's not appropriate.
I didn't delete any categories at all, I just rename them and add more emoticons. Also, I moved some categories to bottom based on usage frequency.
Btw, it can be built now, I will make another pull request.
